test/run_test.py

```diff
@@ -696,6 +777,9 @@ def run_doctests(test_module, test_directory, options):
     "distributed/rpc/test_share_memory": get_run_test_with_subprocess_fn(),
     "distributed/rpc/cuda/test_tensorpipe_agent": get_run_test_with_subprocess_fn(),
     "doctests": run_doctests,
+    "test_ops": run_test_ops,
```

Comment:
Do you think it's possible to make `run_test_ops` generic enough so that we can use it for other test files in the future too?  If yes, it's probably better to choose a generic name like `run_test_in_shards` or something to reflect its nature (not binding to test_ops)

Reply:
one reason it gets its own function is for linalg_cholesky 

the other reason is that test_ops* is usually so large that if it ran serially/did not run in parallel with itself, it would take too long (maybe 2+ hours on windows), while most other tests are small (<1 minute) and wouldnt benefit from getting split up like this